cpu/cc26x0/include/cpu_conf.h

```diff
@@ -49,6 +49,24 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+  *@brief MCU clock configuration
+  * @{
+  */
+
+/* configure HF clock at init by setting 1 for the desired source */
+/* HF clock is used by MCU_VD in active+idle modes and AUX_PD in standby mode */
+#define HF_CLOCK_HF_ROSC 1  /* set 48MHz RCOSC as HF system clock */
+#define HF_CLOCK_HF_XOSC 0  /* set 48MHz XOSC (24MHz doubled internally) as HF system clock */
+
+/* configure LF clock at init by setting 1 for the desired source */
+/* LF clock is available to MCU_VD in standby modes, available to AUX_PD in active mode and is always used by AON */
+#define LF_CLOCK_HF_ROSC  1 /* set 31.25kHz derived from 48MHz RCOSC as LF system clock */
+#define LF_CLOCK_HF_XOSC  0 /* set 31.25kHz derived from 24MHz XOSC as LF system clock */
```

Comment:
even though they are 0 they are defined nonetheless -> `#elif LF_CLOCK_HF_XOSC`would still be taken ;-)